.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,11 @@
 <!--
-Before opening your pull request, have a quick look at our contribution guidelines: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+Before opening your pull request, have a quick look at our contribution guidelines:
+https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+-->
+
+<!--
+Add a preview image of your submission using
```

Comment:
I would reword this using "consider" so  contributors don't feel obliged to use it. For now I would like to keep this as an optional thing so we can get some feedback on the system. Perhaps even ask for feedback here? What do you think @PeterShaggyNoble?

```suggestion
-->

<!--
Consider adding a preview image of your submission using
```